### PR TITLE
reduce plot api imports

### PR DESCRIPTION
**Pull request by archmoj**
Refactoring plot_api files (i.e. excluding `plot_api.js`) to reduce imported functions and help increase code readability.

@etpinard

### CODE REVIEW

**Comment by etpinard** [comment]
To be honest @archmoj, I don't see how this makes our code more _readable_.

I agree our import/export statements could be standardised better, but I think we can wait when we'll make a push for ES6 to spend time on this.

**Comment by etpinard** [comment]
@archmoj ok to close this thing?

**Comment by archmoj** [comment]
> To be honest @archmoj, I don't see how this makes our code more _readable_.
> 
> I agree our import/export statements could be standardised better, but I think we can wait when we'll make a push for ES6 to spend time on this.

Your call. 
Anyway here is one example if one searches for `isPlainObject` in https://github.com/plotly/plotly.js/blob/b6503ac2682bab35e7e6949696a7ce029cba00da/src/plot_api/plot_schema.js there are two ways which is called i.e. with/without `Lib.`.

**Comment by etpinard** [comment]
Ok, I agree the patch you made to `plot_schema.js` is :ok_hand:

File: src/plot_api/helpers.js [hunk 1]
```diff
@@ -12,15 +12,15 @@
 var isNumeric = require('fast-isnumeric');
 var m4FromQuat = require('gl-mat4/fromQuat');
 
-var Registry = require('../registry');
 var Lib = require('../lib');
-var Plots = require('../plots/plots');
-var AxisIds = require('../plots/cartesian/axis_ids');
+var subplotsRegistry = require('../plots/plots').subplotsRegistry;
```
Comment (archmoj) [inline src/plot_api/helpers.js]: The only function used from `Plots` in this file is `subplotsRegistry`.

Comment (etpinard) [reply src/plot_api/helpers.js]: Is there a way to enforce this _rule_ using eslint? If not, than we don't _need_ to do this.

**Pull request merged**